src/public/js/decorators/clipboard.js

```diff
@@ -1,5 +1,5 @@
 module.exports = (node, title = 'Copy to Clipboard', tooltipPlacement = 'top', nodeSelector) => {
-	let clipboard = new Clipboard(node);
+	let clipboard = new ClipboardJS(node);
```

Comment:
This needs an update in the eslint config.

Reply:
Sorry, should have caught this before submitting the PR